packages/react-components/src/button/src/Button.css

```diff
@@ -56,22 +57,22 @@ a.o-ui-button {
 
 /* SOLID */
 .o-ui-button-solid {
-    color: var(--white);
-    background-color: var(--marine-500);
+    color: var(--static-white);
```

Comment:
static? It honnestly start to get confusing.

Reply:
This is not a novel concept, it's a colour that won't bulge based on a theme. Google uses it on Youtube CSS, there is more examples in the wild. How would you make sure a colour doesn't change?

A solid button does not change it's text colour when in dark mode hence it cannot use `--white```. Thoughts?

Reply:
https://spectrum.adobe.com/page/color/#Static-colors see here

Reply:
or here `var(--yt-spec-static-brand-white)` from youtube

Reply:
Well IMO if I see `--white` I know it doesn't change.

If I see `--foreground-color` I know it might change depending on the current theme